Slider/tests/Slider-specs.js

```diff
@@ -58,6 +87,19 @@ describe('Slider', () => {
 		expect(slider).toHaveClass(expected);
 	});
 
+	test('should change value of slider on drag', async () => {
+		render(<Slider activateOnSelect defaultValue={50} />);
+		const slider = screen.getByRole('slider');
+
+		activate(slider);
+		await drag(slider, {delta: {x: 50, y: 0}});
```

Comment:
Interesting!! 

But in my environment the test passes even if I remove this line.
Although aria-valuetext is changed after dragging, it returns a NaN value.
Could you check again? 

We may need to change to clicking instead of dragging.

Reply:
you are right. i put the wrong "unexpectedValue". the initial value is "50 change a value with left right button" , not "50"

Reply:
OK I understand what you intend.
Let's keep current

Reply:
the NaN gave me headaches, to be honest. even with a single click on slider, the aria-valueText becomes NaN. I suspect some calculations of the new value cannot be computed in JSDom?
![image](https://user-images.githubusercontent.com/63335068/191904468-4300de49-9939-4f4a-9d77-5f26a928074b.png)
![image](https://user-images.githubusercontent.com/63335068/191904512-6c83bdfd-74f5-4506-b999-f223a4a1f548.png)

Reply:
We should defer seeing the mystery. :) If you want, you can create a separate issue and see deeper more or throw it to SWP.